extensions/git/src/api/git.d.ts

```diff
@@ -26,6 +26,11 @@ export interface Ref {
 	readonly remote?: string;
 }
 
+export interface Trackingship {
+	readonly local: string;
+	readonly upstarem: string;
```

Comment:
👋 I think you have a typo here
```suggestion
	readonly upstream: string;
```